connectors/sources/salesforce.py

```diff
@@ -1179,113 +1140,106 @@ def map_content_document(self, content_document):
+    def map_salesforce_objects(self, _object):
+        def _format_datetime(datetime_):
+            datetime_ = datetime_ or iso_utc()
+            return datetime.strptime(datetime_, "%Y-%m-%dT%H:%M:%S.%f%z").strftime(
+                "%Y-%m-%dT%H:%M:%SZ"
             )
 
-        case_feeds = case.get("Feeds", []) or []
-        for feed in case_feeds:
-            title = feed.get("Title", "") or ""
-            body = feed.get("Body", "") or ""
-            time_body_pairs.append([feed.get("CreatedDate"), f"{title}\n{body}"])
-
-            feed_comments = feed.get("FeedComments", {}) or {}
-            for comment in feed_comments.get("records", []):
-                time_body_pairs.append(
-                    [comment.get("CreatedDate"), comment.get("CommentBody")]
-                )
-
-        # sort the body values by their associated timestamp
-        time_body_pairs = sorted(
-            time_body_pairs, key=lambda x: "" if x[0] is None else x[0]
-        )
-
-        # ensure string, remove Nones, and remove whitespace
-        bodies = [str(x[-1]).strip() for x in time_body_pairs if x[-1] is not None]
-
-        # finally filter out any empty strings, join and return
-        return "\n\n".join(filter(None, bodies))
-
-    def _collect_case_participant_ids_emails_and_names(self, case):
-        ids = []
-        emails = []
-        names = []
-
-        participants = [
-            case.get("Owner", {}) or {},
-            case.get("CreatedBy", {}) or {},
-        ]
-
-        case_comments = case.get("CaseComments", {}) or {}
-        participants.extend(self._collect_created_by(case_comments))
+        owner = _object.get("Owner", {}) or {}
+        owner_id = owner.get("Id")
+        owner_url = f"{self.base_url}/{owner_id}" if owner_id else ""
 
-        email_messages = case.get("EmailMessages", {}) or {}
-        participants.extend(self._collect_created_by(email_messages))
+        doc = {
+            "_id": _object.get("Id"),
+            "_timestamp": _format_datetime(datetime_=_object.get("LastModifiedDate")),
+            "type": _object.get("attributes", {}).get("type"),
+            "url": f"{self.base_url}/{_object.get('Id')}",
+            "owner": owner.get("Name"),
+            "owner_url": owner_url,
+            "title": _object.get("Name"),
+            "description": _object.get("Description"),
+        }
 
-        for email in email_messages.get("records", []):
-            names.append(email.get("FromName"))
-            emails.append(email.get("FromAddress"))
-            emails.extend(str(email.get("ToAddress", "")).split(";"))
-            emails.extend(str(email.get("CcAddress", "")).split(";"))
-            emails.extend(str(email.get("BccAddress", "")).split(";"))
+        # remove redundant fields from salesforce object
+        for field in ["attributes", "Id", "Name", "Description", "LastModifiedDate"]:
+            if _object.get(field):
+                del _object[field]
```

Comment:
```suggestion
```

We don't want to delete the properties that come from Salesforce. We want to remove the redundant properties that we're defining in addition to what's coming from salesforce.